miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/MiniAppSdkConfig.kt

```diff
@@ -16,7 +16,7 @@ import kotlinx.android.parcel.Parcelize
  * @property miniAppAnalyticsConfigList List of analytic config to send events on.
  * @property sslPinningPublicKeyList List of SSL pinning public keys.
  * @property enableH5Ads Whether the host app wants to enable ad placement beta.
- * @property storageMaxSizeKB Maximum size in KB for each Mini App is allowed to use for Secure Storage, Default is 5MB.
+ * @property maxStorageSizeLimit Maximum size in KB for each Mini App is allowed to use for Secure Storage, Default is 5MB.
```

Comment:
```suggestion
 * @property storageMaxSizeMB Maximum size in MB for each Mini App is allowed to use for Secure Storage, Default is 5MB.
```

Reply:
Addressed.

Reply:
Yeah, I got it. I was concerning about guiding the unit e.g. `MB` for this property.

Reply:
However addressed the review comment and changed to `maxStorageSizeLimitInMB`.

Reply:
Thank you 👍